.github/workflows/lint.yaml

```diff
@@ -35,13 +36,9 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-nodejs-
 
-      - name: Install dependencies
```

Comment:
What about `dependabot` updates for e.g. `prettier`? This should generate new `node_modules`

Reply:
The installation was simply moved into the next step.